src/mscorlib/mscorlib.csproj

```diff
@@ -148,7 +148,6 @@
   <ItemGroup>
     <!-- We want the sources to show up nicely in VS-->
```

Comment:
It is still applicable -- the way we get the sources to show up in VS is by including them in the Compile ItemGroup from the csproj file.